Ludown/README.MD

```diff
@@ -75,24 +75,24 @@ Convert .lu file(s) into LUIS JSON file. Optionally you can also generate a LUIS
 ```bash
 >ludown parse toluis -h
 
-  Usage: ludown parse ToLuis [options]
+  Usage: ludown parse ToLuis --in <luFile> | --lu_folder <inputFolder> [-s]
 
   Convert .lu file(s) into LUIS JSON file. You can optionally also request LUIS batch test input file
 
   Options:
 
-    --in <luFile>                                    .lu file to parse
-    -l, --lu_folder <inputFolder>                    Folder that has the .lu file. By default ludown will only look at the current folder. To look at all subfolders, include -s
-    -o, --out_folder <outputFolder>                  Output folder for all files the tool will generate
-    -s, --subfolder                                  Include sub-folders as well when looking for .lu files
-    --verbose                                        Get verbose messages from parser
-    -r, --luis_schema_version <luis_schema_version>  LUIS Schema version
-    -v, --luis_versionId <luis_versionId>            LUIS app version
-    -n, --luis_name <luis_appName>                   LUIS app name
-    -d, --luis_desc <luis_appDesc>                   LUIS app description
-    -c, --luis_culture <luis_appCulture>             LUIS app culture
-    -t, --write_luis_batch_tests                     Write out LUIS batch test json file
-    -h, --help                                       output usage information
+    --in <luFile>                                               .lu file to parse
+    -l, --lu_folder <inputFolder> [optional]                    Folder that has the .lu file. By default ludown will only look at the current folder. To look at all subfolders, include -s
```

Comment:
msbot and dispatch use the "(optional)" notation right before the description of the parameter.  Perhaps make it consistent?  I don't care either way though. :-)
ie -r, --luis_schema_version <luis_schema_version>   (optional) LUIS Schema version